docarray/array/array/array.py

```diff
@@ -57,7 +58,7 @@ def _delegate_meth(self, *args, **kwargs):
 
 
 class DocumentArray(
-    IndexingSequenceMixin[T_doc], IOMixinArray, AnyDocumentArray[T_doc]
+    IndexingSequenceMixin[T_doc], PushPullMixin, IOMixinArray, AnyDocumentArray[T_doc]
 ):
```

Comment:
Not sure about making Da directly a PushPullMixin. I think this quite polute the namespace of DocumentArray :thinking_face: At the same time I don't have an alternative in mind